packages/renderer/src/lib/kube/details/KubeDeploymentStatusArtifact.svelte

```diff
@@ -23,26 +25,23 @@ export let artifact: V1DeploymentStatus | undefined;
     <tr>
       <Title>Conditions</Title>
     </tr>
-    {#each artifact.conditions as condition}
-      <tr>
-        <Subtitle>{condition.reason}</Subtitle>
-      </tr>
-      <tr>
-        <Cell>Type</Cell>
-        <Cell>{condition.type}</Cell>
-      </tr>
-      <tr>
-        <Cell>Status</Cell>
-        <Cell>{condition.status}</Cell>
-      </tr>
-      <tr>
-        <Cell>Reason</Cell>
-        <Cell>{condition.reason}</Cell>
-      </tr>
-      <tr>
-        <Cell>Message</Cell>
-        <Cell>{condition.message}</Cell>
-      </tr>
-    {/each}
+    <tr>
+      <td colspan="2">
+        <Table>
+          <tr>
+            <th align="left">Type</th><th align="left">Status</th><th align="left">Updated</th><th align="left">Reason</th><th align="left">Message</th>
+          </tr>
+          {#each artifact.conditions as condition}
+            <tr>
+              <td>{condition.type}</td>
+              <td>{condition.status}</td>
+              <td>{humanizeDuration(moment().diff(condition.lastTransitionTime), { round: true, largest: 1 })}</td>
+              <td>{condition.reason}</td>
+              <td>{condition.message}</td>
+            </tr>
+          {/each}
+        </Table>
+      </td>
+    </tr>
```

Comment:
todo: we have the exact same lines in the other file, so create a subcomponent .svelte

and then just call `<StatusTable={artifact.conditions} />`

Reply:
done